html/index.html

```diff
@@ -566,20 +563,20 @@ <h3>Element: Image</h3>
         <h3>Element: Line Break</h3>
           <div>
             <div class="halfblock">
-              <pre><code class="html">&lt;p&gt;
-  Imagine there's no Heaven &lt;br/&gt;
-  It's easy if you try &lt;br/&gt;
-  No hell below us  &lt;br/&gt;
+              <pre><code class="html apache">&lt;div&gt;
```

Comment:
I think we should change this back to `p` because we don't introduce `div` until later

Reply:
Good point, I was conflicted about that
Because it's weird and not great practice to have line breaks inside a p... but we haven't introduced divs yet